packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -81,7 +81,11 @@ class UniswapPriceFeed extends PriceFeedInterface {
     });
 
     // Search backwards through the array and grab block timestamps for everything in our lookback window.
-    const currentTime = this.getTime();
+    // Get Time can either be a synchronous OR asynchronous function depending on how the UniswapPriceFeed is setup.
```

Comment:
This enables the bots clock within this feed to use EITHER the `getTime` synchronous or `getTime` asynchronous methods to accommodate the override in the `CreatePriceFeed.js`.